internal/command/org_metadata_model.go

```diff
@@ -0,0 +1,94 @@
+package command
+
+import (
+	"github.com/zitadel/zitadel/internal/eventstore"
+	"github.com/zitadel/zitadel/internal/repository/org"
+)
+
+type OrgMetadataWriteModel struct {
+	MetadataWriteModel
+}
+
+func NewOrgMetadataWriteModel(orgID, resourceOwner, key string) *OrgMetadataWriteModel {
+	return &OrgMetadataWriteModel{
+		MetadataWriteModel{
+			WriteModel: eventstore.WriteModel{
+				AggregateID:   orgID,
+				ResourceOwner: resourceOwner,
+			},
+			Key: key,
+		},
+	}
+}
```

Comment:
I think orgID and resourceowner is redundant

```suggestion
func NewOrgMetadataWriteModel(orgID, key string) *OrgMetadataWriteModel {
	return &OrgMetadataWriteModel{
		MetadataWriteModel{
			WriteModel: eventstore.WriteModel{
				AggregateID:   orgID,
				ResourceOwner: orgID,
			},
			Key: key,
		},
	}
}
```